vispy/scene/cameras/cameras.py

```diff
@@ -85,7 +76,8 @@ class BaseCamera(Node):
     # These define the state of the camera
     _state_props = ()
 
-    ZOOM_FACTOR = 0.005
+    # The fractional zoom to apply for a single pixel of mouse motion 
+    zoom_factor = 0.007
```

Comment:
I forgot to mention: the example at `basics/scene/sensitivity.py` was added for everyone to try and report their findings. Use the +/- keys to adjust the mouse sensitivity while exploring the data. Report the value you like best as well as your OS, backend, and the input device you were using.
@rossant @Eric89GXL @rougier